src/main/java/util/RemoteUtilities.java

```diff
@@ -0,0 +1,63 @@
+package util;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+public class RemoteUtilities {
+    /**
+     Performs an HTTP request to the specified URL with the given method and JSON input.
+     This method is used internally by the UserDBAccess class.
+     @param method the HTTP method (e.g., GET, POST)
+     @param jsonInputString the JSON input string for POST requests, null for GET requests
+     @return the response from the HTTP request
+     @throws Exception if an error occurs during the HTTP request
+     **/
+    private final String url_base;
+
+    public RemoteUtilities(String url_base) {
+        this.url_base = url_base;
+    }
+
+    public String performHttpRequest(String method, String jsonInputString, Integer id,
```

Comment:
We should definitely try to add some internal documentation here - but before that we should look into breaking this up into more methods as this is almost certainly a [long-method](https://refactoring.guru/smells/long-method) code smell.